app/controllers/coronavirus_landing_page_controller.rb

```diff
@@ -17,4 +17,16 @@ def content_item
   def presenter
     CoronavirusLandingPagePresenter.new(content_item.to_hash)
   end
+
+  def cookie_policy
+    if cookies.has_key?("cookies_policy")
```

Comment:
What's `cookies` here?

Reply:
it's this:
```
pry(#<CoronavirusLandingPageController>)> cookies
=> #<ActionDispatch::Cookies::CookieJar:0x00007f7f41518138
 @committed=false,
 @cookies=
  {"cookies_policy"=>"{\"essential\":true,\"settings\":true,\"usage\":true,\"campaigns\":true}",
   "cookies_preferences_set"=>"true",
   "global_bar_seen"=>"{\"count\":999,\"version\":6}",
   "_ga"=>"GA1.1.1469247533.1585502073",
   "_gid"=>"GA1.1.1336703964.1585502073"},
 @delete_cookies={},
 @request=
  #<ActionDispatch::Request:0x00007f7f416dcac8
   @env= ...
```

Reply:
We can't generally do cookie parsing/behaviour in ruby code because most things are served from Fastly which serves cached html/js/css.  Feels like we might need to do this client side with CSS?

Reply:
ah yes, thanks for explaining.